### PR TITLE
Add setup chapter to readme

### PR DESCRIPTION
**Pull request by gomorizsolt**
Issue #17.

Soon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,27 @@
 
 #### Customizable WebSocket test server in 3 easy steps:
 - [Fork me](https://github.com/c-hive/ws-anywhere/fork)
-- Connect to Heroku (or deploy elsewhere)
+- Deploy somewhere(MongoDB is required)
```
Comment (thisismydesign) [inline README.md]:
```suggestion
- [Deploy me](https://github.com/c-hive/ws-anywhere#deployment)
```

File: README.md [hunk 2]
```diff
@@ -2,9 +2,27 @@
...
 - Set responses on the server's UI
 
+#### Heroku setup
+
+If the target deployment service is Heroku, here is an example script that sets up the environment automatically.
```
Comment (thisismydesign) [inline README.md]:
```suggestion
Use this script te (re)set the app in an idempotent way on Heroku.
```

File: README.md [hunk 3]
```diff
@@ -2,9 +2,27 @@
 
 #### Customizable WebSocket test server in 3 easy steps:
 - [Fork me](https://github.com/c-hive/ws-anywhere/fork)
-- Connect to Heroku (or deploy elsewhere)
+- Deploy somewhere(MongoDB is required)
 - Set responses on the server's UI
 
+#### Heroku setup
```
Comment (thisismydesign) [inline README.md]:
```suggestion
## Deployment
```

I think this could be after the Features chapter.

**Review by thisismydesign — COMMENTED** [review]
Some wording changes.

I think it's fine to only support Heroku in the readme and not go into details about service requirements for different deployments. Others can contribute different guides.

**Comment by gomorizsolt** [comment]
I usually go into the specific details instead of describing the related stuff abstractly. Perhaps this is something that should be improved - just some remarks for myself. :)

**Review by thisismydesign — APPROVED** [review]
I think both approaches can be fine.

**Pull request merged**